src/EnergyPlus/GroundTemperatureModeling/KusudaAchenbachGroundTemperatureModel.cc

```diff
@@ -127,7 +127,7 @@ KusudaGroundTempsModel::KusudaGTMFactory(EnergyPlusData &state, GroundTempObjTyp
                 Real64 maxSurfTemp(-100); // Set low initially but will get updated
 
                 std::shared_ptr<BaseGroundTempsModel> shallowObj = GetGroundTempModelAndInit(
-                    state, state.dataGrndTempModelMgr->CurrentModuleObjects(static_cast<int>(GroundTempObjType::SiteShallowGroundTemp)), "");
+                    state, static_cast<std::string>(groundTempModelNamesUC[static_cast<int>(GroundTempObjType::SiteShallowGroundTemp)]), "");
```

Comment:
All of these should be left as `std::string_view`. If the function requires a `std::string const &` argument, change it to `std::string_view`.